.rubocop_todo.yml

```diff
@@ -425,6 +425,10 @@ Style/EvalWithLocation:
     - 'lib/rmagick_internal.rb'
     - 'lib/rvg/units.rb'
 
+Style/FormatString:
```

Comment:
Why was this excluded? Wouldn't it be easier to follow the style rules?

Reply:
Please see the reason 2 in description. it need to fix the problem.

Reply:
And I don't like this Rubocop's rules for formatting at least.
`format` is just alias of `sprintf` for me.

https://github.com/ruby/ruby/blob/e804fcb42c56d5a4ff31df0391accaf5ede8c26e/object.c#L4181-L4182